src/WebJobs.Extensions.DurableTask/DurableOrchestrationClient.cs

```diff
@@ -90,8 +90,9 @@ public override async Task<string> StartNewAsync(string orchestratorFunctionName
                 throw new ArgumentException($"Instance ID lengths must not exceed {MaxInstanceIdLength} characters.");
             }
 
+            var dedupeStatuses = this.GetStatusesNotToOverride();
             Task<OrchestrationInstance> createTask = this.client.CreateOrchestrationInstanceAsync(
-                orchestratorFunctionName, DefaultVersion, instanceId, input);
+            orchestratorFunctionName, DefaultVersion, instanceId, input, null, dedupeStatuses);
```

Comment:
nit: I don't love the spacing change here. I would recommend putting each parameter on it's own line, at 1+ tab depth. That also gives you room to label parameters that might not be clear (i.e. `null` and `dedupeStatuses` if you rename it).